projects/frontend/data-pipelines/gui/e2e/integration/lib/explore/data-job-details.spec.js

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright 2021-2023 VMware, Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/// <reference types="cypress" />
+import { DataJobsExplorePage } from '../../../support/pages/app/lib/explore/data-jobs.po';
+import { DataJobExploreDetailsPage } from '../../../support/pages/app/lib/explore/data-job-details.po';
+import { applyGlobalEnvSettings } from '../../../support/helpers/commands.helpers';
+
+describe('Data Job Explore Details Page', { tags: ['@dataPipelines', '@exploreDataJobDetails'] }, () => {
+    let dataJobExploreDetailsPage;
+    let testJobs;
+
+    before(() => {
+        return DataJobExploreDetailsPage.recordHarIfSupported()
+                                        .then(() => cy.clearLocalStorageSnapshot('data-job-explore-details'))
+                                        .then(() => DataJobExploreDetailsPage.login())
+                                        .then(() => cy.saveLocalStorage('data-job-explore-details'))
+                                        .then(() => cy.cleanTestJobs())
+                                        .then(() => cy.prepareBaseTestJobs())
+                                        .then(() => cy.fixture('lib/explore/test-jobs.json'))
+                                        .then((loadedTestJobs) => {
+                                            testJobs = applyGlobalEnvSettings(loadedTestJobs);
+
+                                            return cy.wrap({ context: 'explore::data-job-details.spec::before()', action: 'continue' });
+                                        });
+    });
+
+    after(() => {
+        cy.cleanTestJobs();
+
+        DataJobExploreDetailsPage.saveHarIfSupported();
+    });
+
+    beforeEach(() => {
+        cy.restoreLocalStorage('data-job-explore-details');
+
+        DataJobExploreDetailsPage.initBackendRequestInterceptor();
+    });
+
+    it('Data Job Explore Details Page - should load and show job details', () => {
+        cy.log('Fixture for name: ' + testJobs[0].job_name);
+
+        const dataJobsExplorePage = DataJobsExplorePage.navigateTo();
+
+        dataJobsExplorePage
+            .openJobDetails(testJobs[0].team, testJobs[0].job_name);
+
+        dataJobExploreDetailsPage = DataJobExploreDetailsPage
+            .getPage();
+
+        dataJobExploreDetailsPage
+            .getDetailsTab()
+            .should('be.visible');
+
+        dataJobExploreDetailsPage
+            .getMainTitle()
+            .should('be.visible')
+            .should('contains.text', testJobs[0].job_name);
+
+        dataJobExploreDetailsPage
+            .getStatusField()
+            .should('be.visible')
+            // TODO: do the right assertion (once it gets implemented)
```

Comment:
To echo @sabadzhiev , these are pretty useless. Even if I ask whoever wrote this what the right assertion is, they probably don't remember anymore. I vote to remove them.

Reply:
Cleaned up most of the TODOs, some I left for me to revise upon CI/CD e2e execution
cc @sabadzhiev